hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/committer_architecture.md

```diff
@@ -1694,7 +1694,7 @@ must be used, which means: the V2 classes.
 
 **Magic Committer: Name of directory**
```

Comment:
replace with
```
**Magic Committer: Directory Naming**
```
to reflect changes in text

Reply:
ack